soccer/src/soccer/planning/planning_params.cpp

```diff
@@ -0,0 +1,196 @@
+#include "planning_params.hpp"
+
+namespace planning {
+
+DEFINE_NS_FLOAT64(kPlanningParamModule, constraints, max_translational_speed, 2.0,
```

Comment:
I assume there be a good way to set all of this without modifying this code for tuning? Will the comments describing what each thing does have a way to escape this file and be used in that place? I see some of the comments are duplicated in the declaration and in comment form so I guess most of it should be attached to the parameters.